mixins/test/focus-mixin.test.js

```diff
@@ -1,7 +1,7 @@
 
 import { defineCE, expect, fixture, html, oneEvent } from '@open-wc/testing';
 import { FocusMixin } from '../focus-mixin.js';
-import { getComposedActiveElement } from '../../../helpers/focus.js';
+import { getComposedActiveElement } from '../../helpers/focus.js';
```

Comment:
Was `web-dev-server` or `web-test-runner` just kind of... fixing these for us?

Reply:
Not really, it's a browser-level thing. When you try to `..` above the root it does nothing, so you could throw as many `../../../../` in there as you want, it'll always just end up at the root.

When doing rollup, we're not serving the files on the web, so root is no longer project root, it's your file-system's root - which is obviously deeper.

Reply:
Ah, makes sense!